events.html

```diff
@@ -2,13 +2,16 @@
 layout: default
 title: Events
 section_id: events
+cover-pic: /images/group/denero-lilt-cropped.jpg
```

Comment:
Maybe play around with the cropping? Or you can even use the denero-cropped.jpg already in the repository if you don't want to mess with cropping.